src/frontend/planner_test/tests/testdata/output/limit.yaml

```diff
@@ -140,14 +140,15 @@
     create table t (a int);
     select count(*) from t order by 1 limit 1;
   batch_plan: |-
-    BatchTopN { order: [sum0(count) ASC], limit: 1, offset: 0 }
-    └─BatchSimpleAgg { aggs: [sum0(count)] }
-      └─BatchExchange { order: [], dist: Single }
-        └─BatchSimpleAgg { aggs: [count] }
-          └─BatchScan { table: t, columns: [], distribution: SomeShard }
```

Comment:
Not sure why prune TopN columns can also rewrite the BatchTopN 🤔 . Regardless, here it could be rewritten to `BatchLimit` instead I think.

Reply:
Fixed